api-gateway/src/main/resources/application.yml

```diff
@@ -126,3 +126,4 @@ vro:
         url: "https://github.com/department-of-veterans-affairs/abd-vro/blob/master/LICENSE.md"
     servers:
       - url: ${STARTER_OPENAPI_SERVERURL:/}
+      - url: ${VRO_CC_HOSTNAME}:${VRO_CC_PORT}
```

Comment:
This shouldn't be needed since all traffic goes to port 8060
```suggestion
```

The VRO CC port is set here: https://github.com/department-of-veterans-affairs/abd-vro/blob/4c7d466f7c903d5f0ad5efcd80161116b4ba72ce/api-gateway/src/main/resources/application-local.yml#L31